man/frolladapt.Rd

```diff
@@ -0,0 +1,42 @@
+\name{frolladapt}
+\alias{frolladapt}
+\title{Adapt rolling window to irregularly spaced time series}
+\description{
+  Helper function to generate adaptive window size based on the irregularly spaced time series index. Experimental. Generated adaptive window can be then used in rolling functions. See \code{\link{froll}} and \code{\link{frollapply}} for details.
+}
+\usage{
+  frolladapt(x, n, align="right", partial=FALSE, give.names=FALSE)
+}
+\arguments{
+  \item{x}{ Integer. Other objects of type numeric (including \code{Date}, \code{POSIXct} and any others numeric-based class) will be coerced to integer, which, for example, in case of \code{POSIXct} means truncating to whole seconds. Must be sorted, have no duplicate and have no missing values. }
+  \item{n}{ Integer, positive, vector giving rolling window size(s). This is the \emph{total} number of included values in aggregate function. Value corresponds to unit of \code{x}. When \code{x} is a \code{POSIXct} then \code{n} are seconds, when \code{x} is a \code{Date} then \code{n} are days. }
```

Comment:
It's not clear from this description how to interpret when `length(n) > 1`

Reply:
I will make a follow up PR to improve this also for froll.Rd and frollapply.Rd focusing on scalar input in description, and then just at the end mention it can be vectorized.